.rubocop_fixme.yml

```diff
@@ -29,6 +29,7 @@ Metrics/ModuleLength:
     - 'app/models/concerns/hyrax/ability.rb'
     - 'app/services/hyrax/workflow/permission_query.rb'
     - 'spec/services/hyrax/workflow/permission_query_spec.rb'
+    - 'app/models/concerns/hyrax/collection_behavior.rb'
```

Comment:
How about extracting CollectoinAccessControls or something, so we don't have to skip this check?

Reply:
Making this a TODO so we can get the indexing issues resolved ASAP